src/game-engine/application-states/ApplicationStatesPlayerA.js

```diff
@@ -36,6 +36,10 @@ class BasePlayerA {
             stake: this.stake,
         }
     }
+
+    get alwaysSend() {
```

Comment:
why is this called `alwaysSend`?  Why not just `shouldSendMessage`? I guess I'm wondering what this implies... that if `alwaysSend()` returns false, then we should still send a message sometimes?

Reply:
Changed.